### DISCUSSION TITLE
和win10电池自带的性能调节那个有啥区别吗？

### DISCUSSION THREAD

**Discussion by fu7100** [zqhong/RyzenTuner#14] [Q&A]

**Comment by zqhong** [comment]
RyzenTuner 选择不同的模式，最终调整的是 CPU 的功耗限制。对应 [ryzenadj](https://github.com/FlyGoat/RyzenAdj) 的三个参数：

```
    -a, --stapm-limit=<u32>               Sustained Power Limit         - STAPM LIMIT (mW)
    -b, --fast-limit=<u32>                Actual Power Limit            - PPT LIMIT FAST (mW)
    -c, --slow-limit=<u32>                Average Power Limit           - PPT LIMIT SLOW (mW)
```

Windows 10 自带的性能调节具体做了什么不清楚，下面的链接可能有帮助：

* https://docs.microsoft.com/en-us/windows-hardware/customize/desktop/customize-power-slider
* https://github.com/AaronKelley/PowerMode
